parser/feature_pubsub.go

```diff
@@ -30,20 +29,32 @@ func init() {
 		locations.AllowedIn(locations.Variable).ButNotIn(locations.Function),
 	)
 
-	registerResourceUsageParser(
-		est.PubSubTopicResource,
-		"NewSubscription",
-		(*parser).parsePubSubSubscription,
-		locations.AllowedIn(locations.Variable).ButNotIn(locations.Function),
-	)
-
 	registerResourceUsageParser(
 		est.PubSubTopicResource,
 		"Publish",
 		(*parser).parsePubSubPublish,
 		locations.AllowedIn(locations.Function).ButNotIn(locations.InitFunction),
 	)
 
+	registerResource(
+		est.PubSubSubscriptionResource,
+		"pubsub subscription",
+		"https://encore.dev/docs/develop/pubsub",
+		"pubsub",
+		"encore.dev/pubsub",
+		est.PubSubTopicResource,
```

Comment:
this last field means that the Subscription Resource is dependant on the Topic resources being parsed first